bananas_api/helpers/enums.py

```diff
@@ -60,3 +60,27 @@ class Status(Enum):
     OK = "OK"
     WARNINGS = "Warnings"
     ERRORS = "Errors"
+
+
+class Classification(Enum):
+    NEWGRF_AIRCRAFT = "newgrf-aircraft"
+    NEWGRF_AIRPORT = "newgrf-airport"
+    NEWGRF_BRIDGE = "newgrf-bridge"
+    NEWGRF_ECONOMY = "newgrf-economy"
+    NEWGRF_GUI = "newgrf-gui"
+    NEWGRF_INDUSTRY = "newgrf-industry"
+    NEWGRF_LANDSCAPE = "newgrf-landscape"
+    NEWGRF_MIXED = "newgrf-mixed"
+    NEWGRF_OBJECT = "newgrf-object"
+    NEWGRF_RAIL_INFRA = "newgrf-rail-infra"
+    NEWGRF_RAIL_STATION = "newgrf-rail-station"
+    NEWGRF_ROAD_INFRA = "newgrf-road-infra"
+    NEWGRF_ROAD_VEHICLE = "newgrf-road-vehicle"
+    NEWGRF_SHIP = "newgrf-ship"
+    NEWGRF_SIGNAL = "newgrf-signal"
+    NEWGRF_TOWN = "newgrf-town"
+    NEWGRF_TOWNNAME = "newgrf-townname"
+    NEWGRF_TRAIN = "newgrf-train"
+    NEWGRF_VEHICLE = "newgrf-vehicle"
+    NEWGRF_WATER_INFRA = "newgrf-water-infra"
+    UNKNOWN = "unknown"
```

Comment:
For sure the second, as `newgrf-economy` has nothing to do with `gs-economy`.

Really the only exception is `unknown`, which is translated to `Miscellaneous` in the frontend. This is just a generic group for all content that failed to be classified. Which means there wasn't a single feature to base information on.

I suspect NewGRF will be the only category that will have this, but it made programming a lot easier to not assume that. Places like https://github.com/OpenTTD/bananas-api/pull/329/files#diff-e98d17f2790e9deba9ebb60a0eff32d2dad4211574ea1514e520ee2bfbc47604R34. Otherwise you get endless lookup tables to point to an `unknown` category, where you basically alias stuff.

As a positive additional, you can just grep through the code to find all the entries that failed classification. No matter what :)

Don't confuse `unknown` with `unclassified`. `unclassified` is not set, but implied when the key doesn't exist yet. This makes things different between: "we don't know" and "we haven't tried (yet)".
This is mostly to ease up migration (that way we can deploy a new version before the database is updated), and work with categories that don't have a classifier yet.